modules/xmpp/strophe.jingle.js

```diff
@@ -314,7 +314,7 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
 
         switch (action) {
         case 'session-initiate': {
-            logger.log('(TIME) received session-initiate:\t', now);
+            logger.log('Received session-initiate');
```

Comment:
Actually this timestamp was very useful during the last performance testing we were performing. It will be good to have it!

Reply:
How were you using it? I'm genuinely curious, because performance.now() is not a timestamp, so in isolation it doesn't add much! Maybe I can add a metric about start to session-initiate?